Sources/Testing/Events/Event.Recorder.swift

```diff
@@ -344,6 +344,37 @@ extension Event.Recorder {
   }
 }
 
+// MARK: -
+
+extension Difference {
+  /// Get a description of this instance with the given recorder options.
+  ///
+  /// - Parameters:
+  ///   - options: Options to use when writing the comments.
+  ///
+  /// - Returns: A formatted description of `self`.
+  fileprivate func formattedDescription(options: Set<Event.Recorder.Option>) -> String {
+    guard options.contains(.useANSIEscapeCodes) else {
+      return String(describing: self)
+    }
+
+    return String(describing: self)
+      .split(whereSeparator: \.isNewline)
+      .map { line in
+        switch line.first {
+        case "+":
```

Comment:
I'm thinking about refactoring this code a bit in the future to use SF Symbols when available, but I want to factor `_Symbol` out into its own file first, and that's not relevant to this PR.